test/agentchat/test_groupchat.py

```diff
@@ -504,6 +504,19 @@ def test_selection_helpers():
         groupchat.manual_select_speaker()
 
 
+def test_value_error_for_missing_get_socket_client():
+    # Create 3 agents
+    agents = [autogen.agentchat.Agent(name="Agent1"), 
+              autogen.agentchat.Agent(name="Agent2"), 
+              autogen.agentchat.Agent(name="Agent3")]
+
+    with pytest.raises(ValueError, match="get_socket_client_function is required if use_agent_stream is True"):
```

Comment:
Can we also have a test that assures streaming is working?
Also, some example code which instantiates an agent and sets up socket streaming would be helpful.